### PR TITLE
Removing nested project to fix gitignore

### PR DESCRIPTION
**Pull request by Aklaran**
I believe the gitignore issues were caused by the fact that the Unity project was not the root of the repo. I just moved all the files out so the repo root is also the Unity project root. 

This should fix .gitignore issues. Plz test to see that library/cache stuff isn't appearing in git anymore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,5 @@ sysinfo.txt
 # Crashlytics generated file
 crashlytics-build.properties
 
+# MacOS
+.DS_Store
```
Comment (CameronLee97) [inline .gitignore:62]: Is this an auto-gen file specific to MacOS?

**Review by CameronLee97 — CHANGES_REQUESTED** [review]
image



So what's the use of the old DeckBuildingDragon project folder?  It also contains its own Assets and Library folder.  Can we delete this?

File: .gitignore (hunk 1 above)
Comment (Aklaran) [reply .gitignore:62]: Yeah Macs generate this file to store OS-specific info about the containing folder like icon position (if you use the free-floating icon mode).

Comment (Aklaran) [reply .gitignore:62]: For the old DeckBuildingDragon project folder, I think that's left over on your local machine from Unity wanting to re-cache stuff. It doesn't appear in the remote repo as far as I can tell.

I would delete that folder on your machine, and make sure to also remove it from your project list on Unity Hub + add the new root.

Comment (CameronLee97) [reply .gitignore:62]: Ah ok that was it! Thanks, yeah this LGTM.

**Pull request merged**